OneCommunityGlobal/HGNRest
# yihan create submit for review button

# Description
<img width="719" alt="image" src="https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/94303659/4536b0b8-dc49-4dc8-8e9d-04299c0ecd26">

## Related PRS (if any):
This backend PR is related to the [#1013](https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/pull/1013) frontend PR.

## Main changes explained:
- Update file task.js for adding reviewStatus property to task object.

## How to test:
1. check into current branch on both frontend and backend.
2. do `npm install` and `npm run build` to run this PR locally
3. follow the description of [PR#1013](https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/pull/1013)

## Screenshots or videos of changes:

https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/94303659/915fceb5-8325-4cd8-941a-6711d705a33d

## Note:
It may take some time for button to update.
